### ISSUE TITLE
v2.55.0 may break named imports

### ISSUE THREAD

**Issue by dan-serendipity** [mixpanel/mixpanel-js#440]
Prior to v2.55.0, statements like `import { init } from 'mixpanel-browser'` were possible.
The most recent type definitions at `@types/mixpanel-browser` support this.
However, after upgrading to v2.55.0, if your bundler decides to start using `mixpanel.module.js`, you will get a runtime error: `Uncaught SyntaxError: The requested module does not provide an export named 'init'`.
The new esm module has the following export statement only `export { mixpanel as default };`

**Comment by tdumitrescu** [comment]
Hmm I'm not sure how importing `init` worked before, but that was never a documented or supported usage. We don't control the 3rd-party type definitions. The public API has always been that you import the `mixpanel` object.

**Comment by dinesh-kumar-allen** [comment]
Yes post version 2.53 and above ESM tree shaking support for modules is limited and also resulted in bloating of bundle size from 20kB to 40kB (Gzip). 
Bundlephobia link for reference: https://bundlephobia.com/package/mixpanel-browser@2.55.0

**Comment by jakewski** [comment]
Hey @dinesh-kumar-allen, as of 2.54.0 we started bundling the session recording module for the default NPM installation which will account for the increased bundle size. 

We've provided alternatives to import the core SDK by itself (if you do not want to use session replay) or the option to load the session recording module asynchronously - see the NPM installation instructions in the readme https://github.com/mixpanel/mixpanel-js?tab=readme-ov-file#alternative-installation-via-npm.

Hope this helps!
